src/arb/ComplexMat.jl

```diff
@@ -1084,14 +1070,3 @@ function eigenvalues(A::ComplexMat)
    e, _ = _eig_multiple(A)
    return [ x[1] for x in e ]
 end
-
-###############################################################################
```

Comment:
In terms of "things we can probably remove now: also the code in `src/arb` has some methods like e.g. this one:
```julia
function (x::ComplexMatSpace)()
  z = ComplexMat(nrows(x), ncols(x))
  return z
end
```
which could go; similar code is in `RealMat.jl`/`acb_mat.jl`/`arb_mat.jl`.

Oh and also this could (mostly) go:
```julia
for T in [Integer, ZZRingElem, QQFieldElem, Float64, BigFloat, RealFieldElem, ComplexFieldElem, String]
   @eval begin
      function (x::ComplexMatSpace)(y::$T)
         z = x()
         for i in 1:nrows(z)
            for j = 1:ncols(z)
               if i != j
                  z[i, j] = zero(base_ring(x))
               else
                  z[i, j] = y
               end
            end
         end
         return z
      end
   end
end
```
The exception would be the variant taking a `String`. That could be implemented like this (which I am guessing would also be more efficient -- only parse the string once):
```julia
(x::ComplexMatSpace)(y::AbstractString) = x(ComplexFieldElem(y))
```